src/main/io/dashboard.c

```diff
@@ -580,6 +583,33 @@ static void showDebugPage(void)
 }
 #endif
 
+static void showBBPage(void)
```

Comment:
All of this needs to be conditional to `USE_BLACKBOX` or else it will fail on builds without blackbox support.

Reply:
done